versions/latest/modules/en/pages/rancher-admin/scc-registration.adoc

```diff
@@ -0,0 +1,105 @@
+= SUSE Customer Center Registration
+:revdate: 2025-08-19
+:page-revdate: {revdate}
+
+Since Rancher v2.12.1, you should now register your SUSE Rancher Manager Prime Suite with https://scc.suse.com/home[SUSE Customer Center (SCC)] from the Rancher UI. This will enable you to see your registered systems in SCC under your subscription.
+
+== Prequsities
+
+Registering your Rancher Manager instance with SUSE Customer Center requires a SUSE account. If you don't have a SUSE account yet, visit the https://scc.suse.com/[SCC home page] to create one.
```

Comment:
```suggestion
Registering your Rancher Manager instance with SCC requires a SUSE account. If you don't have a SUSE account yet, visit the https://scc.suse.com/[SCC home page] to create one.
```

The acronym should be used since you already defined it and use it for other cases.